docs/_try/logic.md

```diff
@@ -16,28 +28,57 @@ _my-element.js_
 
 {% include project.html folder="try/logic/before" openFile="my-element.js" %}
 
-1. **Add a loop to your template.**
+1.  **Add a boolean property and an array property to your properties getter.**
 
-    We've added an array property, `myArray`, to my-element.js. To loop over `myArray`, add the following code to your template:
+    Replace the static properties getter with the following code:
 
-    ```html
-    <ul>
-      ${this.myArray.map(i => html`<li>${i}</li>`)}
-    </ul>
+    ```js
+    static get properties() {
+      return {
+        message: { type: String },
+        myBool: { type: Boolean },
+        myArray: { type: Array }
+      };
+    }
     ```
 
-2. **Add a conditional to your template.**
+2.  **Initialize the boolean and array properties.**
 
-    We've added a boolean property, `myBool`, to my-element.js. To render conditionally on `myBool`, add the following code to your template:
+    Replace the constructor with the following code:
+
+    ```js
+    constructor() {
+      super();
+      this.message = 'Hello world! From my-element';
+      this.myBool = true;
+      this.myArray = ['an','array','of','test','data'];
+    }
+    ```
 
-    ```html
-    ${this.myBool?
-      html`<p>Render some HTML if myBool is true</p>`:
+3.  **Add a loop to your template.**
+    
+    To loop over the new `myArray` property, add the following code to your template:
+
+    ```js
```

Comment:
this is HTML at the top level here. I think you need to add a `html\`` to make it highlight correctly.